routes/accounts.js

```diff
@@ -13,18 +13,32 @@ const AUTH_OPTIONS = {
   failureRedirect: '/accounts/login'
 }
 
+const createRootItem = Item => user => {
```

Comment:
Note to us: Jrob mentioned in our chat that this functionality should be moved to another file that contains User functions, and out of Routes